docs/integrations/sources/hubspot.md

```diff
@@ -336,7 +336,8 @@ The connector is restricted by normal HubSpot [rate limitations](https://legacyd
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                                          |
 |:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 4.2.20 | 2024-09-21 | [45753](https://github.com/airbytehq/airbyte/pull/45753) | Update dependencies |
+| 4.2.21 | 2024-09-21 | [45753](https://github.com/airbytehq/airbyte/pull/45753) | Update dependencies |
+| 4.2.20 | 2024-09-23 | [42688](https://github.com/airbytehq/airbyte/pull/44899) | Fix incremental search to use primary key as placeholder instead of lastModifiedDate |
```

Comment:
@ehearty can you fix this? I believe it is part of the Vercel check problem. Well still investigating but in anyway this is wrong, also you can use vim to add your line, it will not mess with the file as other IDEs do.

Reply:
Fix this = dates are not descending.

Reply:
@aldogonzalez8 no idea what happened there... but it's fixed now